### PR TITLE
Remove ENABLE_EDX_EXAMS feature flag

### PR DESCRIPTION
**Pull request by annagav**
#### What are the relevant tickets?
Related to #4627

#### What's this PR do?

- removing ENABLE_EDX_EXAMS
- refactoring course status messages
- updating tests for status messages

#### How should this be manually tested?
Nothing should break.

### CODE REVIEW

File: dashboard/utils.py [hunk 1]
```diff
@@ -466,44 +465,19 @@ def get_exam_card_status(self):  # pylint: disable=too-many-return-statements
...
-            if auths.exists():
-                return ExamProfile.PROFILE_SCHEDULABLE
-            else:
-                return ExamProfile.PROFILE_SUCCESS
-
-        if exam_profile.status in (ExamProfile.PROFILE_PENDING, ExamProfile.PROFILE_IN_PROGRESS,):
-            return ExamProfile.PROFILE_IN_PROGRESS
-
-        elif exam_profile.status in (ExamProfile.PROFILE_INVALID, ExamProfile.PROFILE_FAILED,):
-            return ExamProfile.PROFILE_INVALID
```
Comment (umarmughal824) [inline dashboard/utils.py:487]: Don't we need that checks anymore? Is that intentional?

File: dashboard/utils_test.py [hunk 2]
```diff
@@ -836,55 +835,6 @@ def test_get_mmtrack(self):
...
-        [ExamProfile.PROFILE_INVALID, ExamProfile.PROFILE_INVALID, True, True, False, False],
-        [ExamProfile.PROFILE_FAILED, ExamProfile.PROFILE_INVALID, True, True, False, False],
-        ["", ExamProfile.PROFILE_INVALID, True, True, False, True],
-        [ExamProfile.PROFILE_PENDING, ExamProfile.PROFILE_IN_PROGRESS, True, True, False, False],
-        [ExamProfile.PROFILE_IN_PROGRESS, ExamProfile.PROFILE_IN_PROGRESS, True, True, False, False],
-        [ExamProfile.PROFILE_SUCCESS, ExamProfile.PROFILE_SUCCESS, True, True, False, False],
-        [ExamProfile.PROFILE_SUCCESS, ExamProfile.PROFILE_SCHEDULABLE, True, True, True, False],
-    )
-    @ddt.unpack  # pylint: disable=too-many-arguments
-    @patch('dashboard.utils.log')
```
Comment (umarmughal824) [inline dashboard/utils_test.py:851]: It's also associated with the above question.

**Review by umarmughal824 — APPROVED** [review]
working fine and looks good but just a minor concern

File: dashboard/utils.py (hunk 1 above)
Comment (annagav) [reply dashboard/utils.py:487]: The various states of the exam profile are not used anymore because we are not syncing ExamProfiles with Pearson.

Comment (umarmughal824) [reply dashboard/utils.py:487]: okay, then we are good to go.

**Pull request merged**